src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusQueueMessagePumpOptions.cs

```diff
@@ -5,6 +5,8 @@ namespace Arcus.Messaging.Pumps.ServiceBus.Configuration
     /// <summary>
     ///     Options to configure a Azure Service Bus queue <see cref="AzureServiceBusMessagePump"/> implementation.
     /// </summary>
+    /// TODO: remove 'old' workings after the background jobs package is updated with the new messaging package.
+    [Obsolete("Will be removed in the future, please use the " + nameof(AzureServiceBusMessagePumpOptions) + " instead")]
     public class AzureServiceBusQueueMessagePumpOptions : AzureServiceBusMessagePumpOptions
```

Comment:
This is a 1.0 release, please just remove it

Reply:
Removed, resolving.